pay-api/migrations/versions/331c2cdb0b94_fix_payment_account_id_in_payment_table.py

```diff
@@ -0,0 +1,46 @@
+"""fix_payment_account_id_in_payment_table
+
+Revision ID: 331c2cdb0b94
+Revises: 0449261c3fa7
+Create Date: 2020-12-08 09:59:41.540905
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import base64
+
+import sqlalchemy as sa
+from alembic import op
+from flask import current_app
+from sqlalchemy import String, Integer
+from sqlalchemy.sql import column, table
+
+from pay_api.services.oauth_service import OAuthService
+from pay_api.utils.enums import AuthHeaderType, ContentType
+
+# revision identifiers, used by Alembic.
+revision = '331c2cdb0b94'
+down_revision = '0449261c3fa7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Update payment records with null account id."""
+    # 1. Find all payments with account id null
+    # 2. Find the account id from invoice table linking with invoice_reference table.
+    # 3. Update payment with payment account id.
+    conn = op.get_bind()
+    res = conn.execute(f"select id,invoice_number from payment where payment_account_id is null;")
```

Comment:
do we have to match with payment_method as well? does that help?

Reply:
It will just update all records where pay account id is null.